classes/system/heka/aggregator/single.yml

```diff
@@ -0,0 +1,2 @@
+classes:
+- service.heka.aggregator.single
```

Comment:
I guess this breaks the deployment, because the "service.heka.aggregator.single" class does not yet exist when the initial states are applied on the salt master node.

This is for example the error when applying the `reclass` state:

```
oot@cfg01:/usr/share/salt-formulas/env/_formulas/heka# salt-call state.sls reclass
[WARNING ] /usr/lib/python2.7/dist-packages/salt/grains/core.py:1493: DeprecationWarning: The "osmajorrelease" will be a type of an integer.

[ERROR   ] Data passed to highstate outputter is not a valid highstate return: {'local': ['Pillar failed to render with the following messages:', 'Failed to load ext_pillar reclass: ext_pillar.reclass: ']}
```